components/tree/index.jsx

```diff
@@ -118,6 +119,11 @@ class Tree extends React.Component {
 		this.setState({ treeHasFocus: false });
 	}
 
+	handleExpandClick (event, data) {
```

Comment:
One thing that we need to do is to trigger a re-render as mentioned in the comment above.  An good thing is that now I see the branch loading placeholders, whereas before I didn't see them.  I guess we weren't triggering the re-renders for the before.